niri-config/tests/wiki-parses.rs

```diff
@@ -1,19 +1,18 @@
 use std::fs;
 use std::path::PathBuf;
 
-const NO_TEST_COMMENT: &str = "<!-- no-test -->";
-
 struct KdlCodeBlock {
     filename: String,
     code: String,
     line_number: usize,
+    must_fail: bool,
 }
 
 fn extract_kdl_from_file(file_contents: &str, filename: &str) -> Vec<KdlCodeBlock> {
-    // Removes the > from callouts that might contain ```kdl```
-    let lines: Vec<_> = file_contents
+    let mut lines = file_contents
         .lines()
         .map(|line| {
+            // Removes the > from callouts that might contain ```kdl```
             let line = line.trim();
             if line.starts_with(">") {
```

Comment:
But if it starts with "> " then it is not a ">"

Reply:
but then it doesn't remove blank `>` and will cause parser errors

Reply:
What I mean is, replace this part with
```
if line.starts_with("> ") {
    &line[2..]
} else {
    line
}
```
It will work the same as now, and it cannot panic because a two-character starts_with implies length >= 2.

Reply:
That would fail on this:
```md
> ```kdl
> input {
> // some stuff here
>
> // some stuff here thats seperate but still part of input
> ```
```

as this will be converted into
```md
```kdl
input {
// some stuff here
>
// some stuff here thats seperate but still part of input
```

as the middle > doesn't have a space in front of it. This breaks parsing.

Reply:
Ah. That's right. Thanks for sticking with me